src/components/listView/ListView.styles.js

```diff
@@ -1,11 +1,9 @@
 import {StyleSheet} from 'react-native';
+import {size} from '../../theme';
```

Comment:
Will this include the amount of padding?

Reply:
yes